src/lib.rs

```diff
@@ -104,6 +104,9 @@ pub extern fn update_state(elapsed_time: f32) {
         Some(x) => state.enemies.push(x),
         None => {},
     }
+
+    let stage_height: u16 = state.height;
```

Comment:
not great but I can't figure out how to fix `error[E0502]: cannot borrow `*state` as immutable because it is also borrowed as mutable` :/